### PR TITLE
New Persons Page

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

- Implements the new persons page (partial of #2353), everything behind the `persons-2353` feature flag.

**Default view**

**View identified users only**

**Insights / trends view**

- Adds `is_identified` attribute to PersonSerializer (Postgres)
- Bug fix when searching by properties on persons page (Postgres-only)

## Checklist

- [X] All querysets/queries filter by Organization, by Team, and by User
- [X] Django backend tests
- [X] Jest frontend tests
- [X] Cypress end-to-end tests

### CODE REVIEW

File: posthog/api/person.py
Comment (macobo) [inline posthog/api/person.py:100]: Context on this change?

File: frontend/src/scenes/persons/personsLogic.ts
Comment (macobo) [inline frontend/src/scenes/persons/personsLogic.ts:29]: Nitpick: This is hard to follow. Suggestion:

```
const match = persons && persons.results.find((person) => person.properties?.email)

return match?.properties?.email || 'example@gmail.com
```

**Review by macobo — APPROVED** [review]
Needs a rebase, otherwise lgtm

File: posthog/api/person.py
Comment (paolodamico) [reply posthog/api/person.py:100]: It's just to make sure that we only treat the has_key behaviour if the search term is actually `has:X`, to avoid weird edge cases on trying to use different filters that we do not support yet.

**Comment by paolodamico** [comment]
Thanks for the review! Looks like #2576 was merged and I need to update this to consider the new cohorts structure. Will update and tag you for another quick review.

**Comment by paolodamico** [comment]
Ready for another quick look @macobo, FYI I'll probably address #2685 after this is merged because the new persons table looks very cluttered with the new cohorts experience.

**Review by macobo — APPROVED** [review]
Did not click through it again, but everything makes sense!

**Pull request merged**
